modules/enabling-entra-workload-id-existing-cluster.adoc

```diff
@@ -0,0 +1,240 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-entra-workload-id-existing-cluster"]
+= Enabling {entra-first} on an existing cluster
+
+If you did not configure your {azure-first} {product-title} cluster to use {entra-first} during installation, you can enable this authentication method on an existing cluster.
+
+[IMPORTANT]
+====
+This process is disruptive and takes a significant amount of time.
+Before proceeding, review the following steps and ensure that you accept the time requirement.
+====
```

Comment:
@jeana-redhat These are two known issues that we should also mention in the documentation. 
1. This migration process involves refreshing all ServiceAccounts and restarting all pods, which causes disruption to the customer's workload. Our suggestion is for the client to temporarily halt these services, and then redeploy them after the migration has completed.
2. Don’t trigger the upgrade process during migration. If an upgrade is triggered, the migration will fail.

@jianping-shu  I have summarized two known issues. Please help check if there are any other issues that need attention, when you have time. Thanks.

Reply:
Thanks! I will add these warnings at the beginning